src/content.rs

```diff
@@ -192,12 +192,15 @@ impl Content {
 impl FromStr for Content {
     type Err = anyhow::Error;
     fn from_str(full_document: &str) -> Result<Self, Self::Err> {
-        let (toml_text, body) = full_document
+        // This is a heavy-handed way of normalizing the document to only use "\n"
+        // for line endings. This simplifies a number of things, including Rhai scripting.
+        let doc = full_document.replace("\r\n", "\n");
+        let (toml_text, body) = doc
             .split_once(DOC_SEPARATOR)
-            .unwrap_or(("title = 'Untitled'", full_document));
+            .unwrap_or(("title = 'Untitled'", &doc));
         let head: Head =
             toml::from_str(toml_text).map_err(|e| anyhow::anyhow!("TOML parsing error: {}", e))?;
 
         Ok(Content::new(head, body.to_owned()))
     }
-}
+}
```

Comment:
`cargo fmt` complains about the missing new line at the end, so running it should fix CI here.